common/src/app/common/files/changes.cljc

```diff
@@ -425,7 +425,12 @@
       [:type [:= :set-token]]
       [:set-name :string]
       [:token-name :string]
-      [:token [:maybe ctob/schema:token-attrs]]]]]])
+      [:token [:maybe ctob/schema:token-attrs]]]]
+
+    [:mod-base-font-size
```

Comment:
Sorry, I forgot commenting this. We are tending to use `set-` better than `mod-` in changes (also if you make the change `set-base-font-size` it equals the name of the `ctf` function.